test/components/manager_vault/_hooks.mjs

```diff
@@ -0,0 +1,58 @@
+import { address, randomSeed } from '@waves/ts-lib-crypto';
+import {
+  data,
+  massTransfer,
+  nodeInteraction,
+} from '@waves/waves-transactions';
+import { create } from '@waves/node-api-js';
+import { format } from 'path';
+import { setScriptFromFile } from '../../utils/utils.mjs';
+
+const { waitForTx } = nodeInteraction;
+const apiBase = process.env.API_NODE_URL;
```

Comment:
can be imported from [api.mjs](https://github.com/waves-exchange/contracts/blob/main/test/utils/api.mjs#L4)

Reply:
Changed in all tests